Soma-Technologies-Inc/visitAfrica-backend
# #175494078  Avail requests for approval

#### What does this PR do?

- avail requests for approval by requester's manager

#### Description of Task to be completed?

- the created trip requests should be available to their requester's manager for for being approved

#### How should this be manually tested?

- open your terminal and clone this repo by running git clone https://github.com/Soma-Technologies-Inc/visitAfrica-backend

- run git checkout ft-request-avail-175494078  to navigate to my branch

- run git pull to make sure you have updated code

- run npm install to install all needed packages

- run npm run migrate to create migrations for the databas

- navigate to login route POST /api/auth/login and login to get token that will be used as a header for getting created requests as line manager

- use these body data { "email": "mukamana@gmail.com", "password":"mukamana@Q123" }

- you will get a jwt then

- navigate to get request route GET `http://localhost:3000/api/request/trip-request`

- before making a request append a Authorization header which have value bearer token replace token with the jwt token you got from login route

#### Screenshots (if appropriate)
![Screenshot from 2020-12-27 12-32-56](https://user-images.githubusercontent.com/70438710/103168864-be6ebd00-483f-11eb-8bf5-fd8078e4b117.png)
